app/views/quotes/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h3>Quotes Found:</h3>
+<p>keyword: <%= @keyword %></p>
+<p>results found: <%= @quotes.count %></p>
```

Comment:
Even though this might be more than 10, I believe the API has a default of how many quotes per page that it returns. For example, there could be 120 quotes, but the api delivers that on 6 different pages where each page has 20 quotes. By default, you only get the first page of the api, so you can't be sure that the amount of quotes on the first page that's returned is the total number of quotes that are returned for that keyword search.

See my above comments in regards to getting that total count from the api response.